main.py

```diff
@@ -133,6 +125,18 @@ def get_jenkins_jobs(server, job_search_fields):
 
 	return relevant_jobs
 
+def get_bugs_set(blockers_file):
+    # Generates a set from the entire list of bugs
```

Comment:
```suggestion
    ''' Takes in blockers object and generates a set of all unique bug ids including 0 if it is present
    '''
```
More pythonic, will be good for future documentation